content/millennium/r4/other/health-cards.md

```diff
@@ -0,0 +1,80 @@
+---
+title: Health Cards | R4 API
+---
+
+# Health-Cards
+
+* TOC
+{:toc}
+
+## Overview
+
+A Health Card is a verifiable, digital artifact that represents a limited set of information about an individual, relevant to a specific health status. The card can be presented in digital and physical form. A QR code representing the card can be shared from a device or a piece of paper. Information in the health card can be used to determine the authenticity of the artifact and its issuer. The goal is to provide individuals with the choice to share limited health information in a portable and secure manner.
```

Comment:
Reading the description in this context - do we need to specify that this API is returning the JWS form of the health card and NOT a QR code?

Reply:
How about this? https://github.com/cerner/fhir.cerner.com/pull/659/commits/010373b4f33500144ed55e579fc13e86d741d51a#diff-bf1e7917ef250f45c2b8a4a1b46b31dc38eb42ee5dd2957211f0ba40692d42b1R16

Reply:
I think that's more-clear. I more have a punctuation nit:

> This FHIR $health-cards-issue Operation returns the Health Card as JWS in a FHIR Parameter resource. The response adheres to the [FHIR $health-cards-issue Operation Specification](https://spec.smarthealth.cards/#via-fhir-health-cards-issue-operation).

Reply:
Addressed here https://github.com/cerner/fhir.cerner.com/pull/659/commits/9a98a7942e70bcc47013fbfbd75a8d35cfc6b1d3#diff-bf1e7917ef250f45c2b8a4a1b46b31dc38eb42ee5dd2957211f0ba40692d42b1R16